demos/cat-api/src/main/resources/application.yml

```diff
@@ -12,7 +12,7 @@ gnirps:
             description: "Just your average example of API using cats"
             version: 1.2.0-SNAPSHOT
     database:
-        hbm2ddl-auto: update
+        hbm2ddl-auto: create
```

Comment:
question(non-blocking): Why do you update the value from `update` to `create`?

Reply:
Considering the low importance of the question and the more pressing matters, I'll resolve the conversation. Feel free to re-open it of issue a ticket if need be.